Apps/Common/PiolaOperators.h

```diff
@@ -0,0 +1,140 @@
+//==============================================================================
+//!
+//! \file PiolaOperators.h
+//!
+//! \date Apr 30 2024
+//!
+//! \author Arne Morten Kvarving / SINTEF
+//!
+//! \brief Various weak, discrete Piola-mapped operators.
+//!
+//==============================================================================
+
+#ifndef PIOLA_OPERATORS_H_
+#define PIOLA_OPERATORS_H_
+
+class FiniteElement;
+class Tensor;
+class Vec3;
+
+#include "MatVec.h"
+#include "EqualOrderOperators.h"
+
+/*! \brief Common operators using Piola mapped discretizations.
+ *  \details The operators use the block ordering used in the BlockElmMats class.
+ */
+
+class PiolaOperators
+{
+public:
+  //! \brief Common weak operators using Piola mapped discretizations.
+  class Weak
+  {
+  public:
```

Comment:
Isn't it better and more clear to use nested namespaces instead of classes here? Since they only contain static methods anyway.

Reply:
I need the types for templating purposes.